filters/fadein/fadein.go

```diff
@@ -0,0 +1,252 @@
+package fadein
+
+import (
+	"fmt"
+	"net"
+	"net/url"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zalando/skipper/eskip"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/routing"
+)
+
+const (
+	DurationName        = "fadeInDuration"
+	EaseName            = "fadeInEase"
+	EndpointCreatedName = "endpointCreated"
+)
+
+type (
+	duration time.Duration
+	ease     float64
```

Comment:
Is `ease` used to wait before starting a fade-in to let JIT compilers do their work before we start to fade-in traffic?

Reply:
No, see: https://github.com/zalando/skipper/pull/1447/files#diff-8ef30ff734de1bf682a3b0d530f9f663R64

Reply:
Ok, JIT apps can also use startupProbe, but `ease` in its use: y in this function: `f(x,y) : x ** y`